README.md

```diff
@@ -367,6 +367,8 @@ The following assumes you've completed the steps for [Setting up ExecuTorch](#se
 #### Requirements
 - [Xcode](https://apps.apple.com/us/app/xcode/id497799835?mt=12/) 15.0 or later
 - [Cmake](https://cmake.org/download/) 3.19 or later
+  - Download and open macOS 10.13 or later `.dmg` installer and move the Cmake app to `/Applications` folder.
```

Comment:
Why does it matter that one downloads one Dmg vs another? Realistically, if one targets M1 mac, every universal binary actually targets MacOS-11 or later

Reply:
@malfet it's just what it's named on the download page of the official website :) Let's skip being specific if it sounds confusing,